apps/volunteer/shifts/talks.py

```diff
@@ -0,0 +1,152 @@
+from ..event_date import edt
+
+herald_pattern = [
+            {
+                "first": edt("fri", "09:00:00"),
+                "final": edt("fri", "21:00:00"),
+                "min": 1,
+                "max": 1,
+            } for d in ["fri", "sat", "sun"]
+        ]
+
+stage_a_pattern = [
+    {"first": edt(d, "09:45:00"),
+     "final": edt(d, "12:00:00"),
+     "min": 1,
+     "max": 1,
+     "base_duration": 135
+    } for d in ["fri", "sat"]
+] +[
+    {"first": edt(d, "12:00:00"),
+     "final": edt(d, "20:00:00"),
+     "min": 1,
+     "max": 1,
+    } for d in ["fri", "sat"]
+] + [
+    {"first": edt("sat", "09:45:00"),
+     "final": edt("sat", "12:00:00"),
+     "min": 1,
+     "max": 1,
+     "base_duration": 135
+    },
+    {"first": edt("sat", "12:00:00"),
+     "final": edt("sat", "20:00:00"),
+     "min": 1,
+     "max": 1,
+    },
+    {"first": edt("sun", "09:45:00"),
+     "final": edt("sun", "17:45:00"),
+     "min": 1,
+     "max": 1,
+    },
+    {"first": edt("sun", "17:45:00"),
+     "final": edt("sun", "19:00:00"),
+     "min": 1,
+     "max": 1,
+     "base_duration": 135
+    }
+]
+
+stage_bc_pattern = [
+    {"first": edt("fri", "10:45:00"),
+     "final": edt("fri", "17:45:00"),
+     "min": 1,
+     "max": 1,
+     "base_duration": 140
+    },
+    {"first": edt("fri", "17:45:00"),
+     "final": edt("fri", "20:00:00"),
+     "min": 1,
+     "max": 1,
+     "base_duration": 135
+    },
+    {"first": edt("sat", "09:45:00"),
+     "final": edt("sat", "12:00:00"),
+     "min": 1,
+     "max": 1,
+     "base_duration": 135
+    },
+    {"first": edt("sat", "12:00:00"),
+     "final": edt("sat", "20:00:00"),
+     "min": 1,
+     "max": 1,
+    },
+    {"first": edt("sun", "09:45:00"),
+     "final": edt("sun", "16:30:00"),
+     "min": 1,
+     "max": 1,
+     "base_duration": 135
+    },
+    {"first": edt("sun", "16:30:00"),
+     "final": edt("sun", "18:30:00"),
+     "min": 1,
+     "max": 1,
+    }
+]
```

Comment:
VM should be 2 shifts per day (with 30 min overlap)
Other stage volunteers should be 4 shifts per day (with 15 min overlap)